gradle-baseline-java/build.gradle

```diff
@@ -61,6 +61,7 @@ tasks.test.dependsOn tasks.publishToMavenLocal
 test {
     environment 'CIRCLE_ARTIFACTS', "${buildDir}/artifacts"
     environment 'CIRCLE_TEST_REPORTS', "${buildDir}/circle-reports"
+    systemProperty 'ignoreDeprecations', 'true'
```

Comment:
we discussed this internally some time ago - I actually want to set it on everything:

> should we just ignoreDeprecations for every gradle project? Some reasons why:
> 1. Deprecations often come from other repos we end up running as part of tests. For example, in <internal-repo> there are some tests that run tasks from <other-internal-repo>, that then fail with deprecations warnings due to <other-internal-repo>'s tasks. Since we can’t fix them in the <internal-repo>, we end up adding `--warning-mode=none` to every test, losing deprecation warnings anyway. Or worse, people add `System.setProperty('ignoreDeprecations', 'true')` to individual tests as nebula suggests, making the build flake.
> 2. It’s impossible to see what caused deprecation warnings using nebula-test. The only way I found out to get good info is to add the buildscan plugin to the test.
> 3. We don’t tend to fix deprecations until a major version remove the feature anyway.
> 4. This leads to our gradle plugin repos being super out of date for their own gradle wrapper, meaning they get wrecked by stuff like not supporting jackson 2.15.2's java 20 code.

In this case, we get deprecation warnings about GCV not working with Gradle 9 when running in 8.3/8.4 - something we can't even fix in this repo.